### PR TITLE
refactor:  rebrand to axe-core-maven-html

### PR DESCRIPTION
**Pull request by AdnoC**
Takes https://github.com/dequelabs/axe-core-maven-html/pull/64 and makes changes so that it is usable with axe-devtools-maven-html

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [ ] Code is reviewed for security

### CODE REVIEW

**Comment by stephenmathieson** [comment]
Please fix the conflict.

File: src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+   */
+  public static Results analyze(final WebDriver webDriver,
+      final WebElement context)
+      throws IOException, OperationNotSupportedException {
+    if (webDriver == null) {
+      throw new NullPointerException("the webDriver is null");
+    }
+
+    if (context == null) {
+      throw new NullPointerException("the webDriver is null");
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java:84]:
```suggestion
      throw new NullPointerException("the context is null");

```

File: src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   */
+  public static Results analyze(final WebDriver webDriver,
+      final WebElement context, final AxeBuilderOptions axeBuilderOptions)
+      throws OperationNotSupportedException, IOException {
+    if (webDriver == null) {
+      throw new NullPointerException("the web Driver is null");
+    }
+
+    if (context == null) {
+      throw new NullPointerException("the web Element is null");
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java:107]: Should this be "the web Element" or "the context"?

File: src/main/java/com/deque/html/axecore/results/Platform.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    public void setTestMachine(String testMachine) {
+        this.testMachine = testMachine;
+    }
+
+
+
+    /**********************************
+     * Below is generated code.
+     * DO NOT MODIFY
+     * Regenerate instead
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/results/Platform.java:28]: How do we "regenerate" this?

File: src/main/java/com/deque/html/axecore/results/Rule.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    public void setCreatedDate(String createdDate) {
+        this.createdDate = createdDate;
+    }
+
+    /**********************************
+     * Below is generated code.
+     * DO NOT MODIFY
+     * Regenerate instead
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/results/Rule.java:94]: How do we generate this?

File: src/main/java/com/deque/html/axecore/results/TestResults.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    public void setEndTime(String endTime) {
+        this.endTime = endTime;
+    }
+
+
+
+    /**********************************
+     * Below is generated code.
+     * DO NOT MODIFY
+     * Regenerate instead
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/results/TestResults.java:73]: How?

File: src/main/java/com/deque/html/axecore/results/TestSubject.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        } else {
+            return null;
+        }
+    }
+
+
+    /**********************************
+     * Below is generated code.
+     * DO NOT MODIFY
+     * Regenerate instead
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/results/TestSubject.java:44]: How?

File: src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java [hunk 7]
```diff
@@ -0,0 +1,442 @@
...
+    validateNotNullParameter(webDriver);
+    String rawOptionsArg = getOptions().equals("{}")
+        ? AxeReporter.serialize(runOptions) : getOptions();
+    Object[] rawArgs = new Object[] {rawContextArg, rawOptionsArg};
+
+    if (injectAxe) {
+      try {
+        WebDriverInjectorExtensions.inject(
+            webDriver, builderOptions.getScriptProvider());
+      } catch (IOException e) {
```
Comment (stephenmathieson) [inline src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java:391]: Why catch two different exceptions but do the same thing for both?

File: src/main/java/com/deque/html/axecore/results/Platform.java (hunk 3 above)
Comment (AdnoC) [reply src/main/java/com/deque/html/axecore/results/Platform.java:28]: You'd have to ask the one who wrote attest-java-reporter, these files were copied from there. Not sure what IDE they used, but we can probably manually edit the equals, hashCode, and toString methods if we need to.

File: src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java (hunk 7 above)
Comment (AdnoC) [reply src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java:391]: Good point. Even if an unexpected error occurs we probably still want to tell the user that it was a problem with injection.

**Comment by AdnoC** [comment]
Rebased off of develop to resolve conflict

**Pull request merged**